logger/README.md

```diff
@@ -1,4 +1,4 @@
-# Logger
+# Package `logger` v1.0.0
```

Comment:
There is a `logger/v7.1.0` tag, should we keep it or remove it ?

Reply:
I think the tag can safely be removed.

Reply:
Okay, I'll delete it :)